garmin_txt.cc

```diff
@@ -711,15 +711,15 @@ track_disp_wpt_cb(const Waypoint* wpt)
   }
 
   *fout << "\t";
-  double depth = WAYPT_GET(wpt, depth, unknown_alt);
+  double depth = (wpt->depth_value_or(unknown_alt));
   if (depth != unknown_alt) {
     print_distance(depth, 1, 0, 1);
   }
 
   if (prev != nullptr) {
     *fout << "\t";
     delta = wpt->GetCreationTime().toTime_t() - prev->GetCreationTime().toTime_t();
-    float temp = WAYPT_GET(wpt, temperature, -999);
+    float temp = (wpt->temperature_value_or(-999));
     if (temp != -999) {
       print_temperature(temp);
     }
```

Comment:
shouldn't the printing of temperature, and it's tab, be outside the if statement?